### PR TITLE
create basic mvn project structure

### PR DESCRIPTION
**Pull request by holounic**
Create maven project structure
Add initial dependencies for maven webapp

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.google.sps</groupId>
+    <artifactId>step240-2020</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+
+    <dependencies>
```
Comment (MagoDelBlocco) [inline pom.xml:12]: Consider adding JUnit or another testing framework as a dependency

Comment (holounic) [reply pom.xml:12]: I think, it would be better if we add necessary dependencies only when we actually need them

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                <version>2.2.0</version>
+                <configuration>
+                    <!-- TODO: set project ID. -->
+                    <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+                    <deploy.version>1</deploy.version>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (MagoDelBlocco) [inline pom.xml:49]: Add a [newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) at the end of each file

**Comment by MagoDelBlocco** [comment]
LGTM

File: target/step240-2020-1.0/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+    <threadsafe>false</threadsafe>
+    <sessions-enabled>true</sessions-enabled>
+    <runtime>java8</runtime>
+    <static-files>
+        <!-- prevent unwanted caching when accessing via the web preview server -->
+        <include path="/**" expiration="0s" />
+    </static-files>
+</appengine-web-app>
```
Comment (Somae) [inline target/step240-2020-1.0/WEB-INF/appengine-web.xml:10]:
```suggestion
</appengine-web-app>

```

**Comment by Somae** [comment]
Please convert all tab sizes in all files to two spaces before submitting!

**Pull request merged**